cypress/integration/examples/cypress_api.spec.js

```diff
@@ -0,0 +1,219 @@
+/// <reference types="cypress" />
+
+context('Cypress.Commands', () => {
```

Comment:
The test description does not match the task requirements. The task requires testing the login, add book, and delete book functionality of the Book Store app.